test/passing/module.ml

```diff
@@ -28,3 +28,41 @@ module C = struct
 
   and B : sig end = B
 end
+
+let _ =
+  ( module M : S
+    with type long_long_type = t
```

Comment:
This seems to be just wrong, `with` modifies `S`, not `module M`. Am I missing something?

Reply:
I agree, I added indentation